docs/applications/timemachine.md

```diff
@@ -0,0 +1,24 @@
+# Time Machine
+
+Apple docs: [https://support.apple.com/en-us/HT201250](https://support.apple.com/en-us/HT201250)
+Docker image: [https://github.com/mbentley/docker-timemachine](https://github.com/mbentley/docker-timemachine)
+
+Time Machine is an application that allows you to backup files from you Mac.
```

Comment:
Please can you fix the typo in this line

Reply:
Done.